src/services/api/notification-recipients/notification.recipients.service.ts

```
@@ -447,14 +447,13 @@ export class NotificationRecipientsService {
-      case NotificationEvent.SPACE_COMMUNITY_NEW_MEMBER_ADMIN:
-      case NotificationEvent.SPACE_COLLABORATION_POST_CREATED_ADMIN:
-      case NotificationEvent.SPACE_COLLABORATION_POST_CREATED:
-      case NotificationEvent.SPACE_COLLABORATION_POST_COMMENT_CREATED:
-      case NotificationEvent.SPACE_COLLABORATION_WHITEBOARD_CREATED:
-      case NotificationEvent.SPACE_COMMUNICATION_MESSAGE_RECIPIENT:
+      case NotificationEvent.SPACE_ADMIN_COMMUNITY_APPLICATION:
+      case NotificationEvent.SPACE_ADMIN_COMMUNITY_NEW_MEMBER:
+      case NotificationEvent.SPACE_ADMIN_COLLABORATION_CALLOUT_CONTRIBUTION:
+      case NotificationEvent.SPACE_COLLABORATION_CALLOUT_CONTRIBUTION:
+      case NotificationEvent.SPACE_COLLABORATION_CALLOUT_POST_CONTRIBUTION_COMMENT:
+      case NotificationEvent.SPACE_COLLABORATION_CALLOUT_CONTRIBUTION:
```

Biome (2.1.2)

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)